third_party/perfetto/METADATA

```diff
@@ -1,3 +1,22 @@
+name: "perfetto"
+description:
+  "Perfetto system profiling and tracing"
+
 third_party {
+  identifier {
+    type: "ChromiumVersion"
+    value: "114.0.5735.338"  # from https://chromereleases.googleblog.com/2024/03/long-term-support-channel-update-for.html
+  }
+  identifier {
+    type: "Git"
+    value: "https://android.googlesource.com/platform/external/perfetto.git"
+    version: "f2da6df2f144e41e1c1428f11e8b388eaf8a2209"
+    # from https://chromium.googlesource.com/chromium/src/+/refs/tags/114.0.5735.338/DEPS#1698
```

Comment:
perhaps
https://chromium.googlesource.com/chromium/src/+/114.0.5735.355/DEPS#1698

Reply:
Updated to shorter link, but left hte 388 in there - unless there's a strong reason to pick older ?

Reply:
Perhaps this explains the confusion: 338 looks like 388.

Reply:
Updated to `114.0.5735.356`, i.e. latest tag available - the imported hash doesn't change.